gradle-versions/src/main/java/com/markelliot/gradle/versions/UpdatePluginsTask.java

```diff
@@ -52,9 +52,9 @@ private void setNewPluginVersions(File sourceFile, Map<String, String> pluginUpd
     static String applyUpdate(String content, String pluginName, String pluginVersion) {
         Pattern pattern =
                 Pattern.compile(
-                        "id(\\s+|\\s*\\()(\"|')"
+                        "id(\\s+|\\s*\\()([\"'])"
                                 + Pattern.quote(pluginName)
-                                + "(\"|')(\\s+|\\)\\s+)version\\s+(\"|')([^\"]+)(\"|')");
+                                + "([\"'])(\\s+|\\)\\s+)version\\s+([\"'])([^\"']+)([\"'])");
```

Comment:
I got suggestion to swap `(\"|')` to a group match `[\"']`, the issue was that `([^\"']+)` didn't include `'`

Reply:
Ah, makes sense. Not sure why the tests passed … care to add one that covers this?